scripts/generate-badges.ps1

```diff
@@ -34,6 +34,26 @@ function Get-AzureLine($branchName, $jobNames, [switch]$integration = $false) {
     return $line + [Environment]::NewLine
 }
 
+function Get-BuildsTable() {
+    $jobNames = @(
+        'Build_Windows_Debug'
+        'Build_Windows_Release'
+        'Build_Unix_Debug'
+    )
+
+    $table = @'
+### Builds
+|Branch|Windows Debug|Windows Release|Unix Debug|
```

Comment:
nit: Some markdown parsers (e.g. kramdown) require a blank line around headings. See [MD022](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).